api/src/paths/alert/{alertId}/index.ts

```diff
@@ -0,0 +1,310 @@
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { SYSTEM_ROLE } from '../../../constants/roles';
+import { getDBConnection } from '../../../database/db';
+import { systemAlertSchema } from '../../../openapi/schemas/alert';
+import { authorizeRequestHandler } from '../../../request-handlers/security/authorization';
+import { AlertService } from '../../../services/alert-service';
+import { getLogger } from '../../../utils/logger';
+
+const defaultLog = getLogger('paths/alert/{alertId}/index');
+
+export const GET: Operation = [
+  authorizeRequestHandler(() => {
+    return {
+      and: [
+        {
+          discriminator: 'SystemUser'
+        }
+      ]
+    };
+  }),
+  getAlertById()
+];
+
+GET.apiDoc = {
+  description: 'Gets a specific system alert.',
+  tags: ['alerts'],
+  security: [
+    {
+      Bearer: []
+    }
+  ],
+  parameters: [
+    {
+      in: 'path',
+      name: 'alertId',
+      schema: {
+        type: 'string'
+      }
+    }
+  ],
+  responses: {
+    200: {
+      description: 'System alert response object',
+      content: {
+        'application/json': {
+          schema: systemAlertSchema
+        }
+      }
+    },
+    400: {
+      $ref: '#/components/responses/400'
+    },
+    401: {
+      $ref: '#/components/responses/401'
+    },
+    403: {
+      $ref: '#/components/responses/403'
+    },
+    500: {
+      $ref: '#/components/responses/500'
+    },
+    default: {
+      $ref: '#/components/responses/default'
+    }
+  }
+};
+
+/**
+ * Get a specific system alert by its id
+ *
+ * @returns {RequestHandler}
+ */
+export function getAlertById(): RequestHandler {
+  return async (req, res) => {
+    defaultLog.debug({ label: 'getAlertById' });
+
+    const connection = getDBConnection(req.keycloak_token);
+
+    try {
+      await connection.open();
+
+      const alertId = Number(req.params.alertId);
+
+      const alertService = new AlertService(connection);
+
+      const alert = await alertService.getAlertById(alertId);
+
+      await connection.commit();
+
+      return res.status(200).json(alert);
+    } catch (error) {
+      defaultLog.error({ label: 'getAlertById', message: 'error', error });
+      await connection.rollback();
+      throw error;
+    } finally {
+      connection.release();
+    }
+  };
+}
+
+export const PUT: Operation = [
+  authorizeRequestHandler(() => {
+    return {
+      and: [
+        {
+          discriminator: 'SystemUser',
+          validSystemRoles: [SYSTEM_ROLE.SYSTEM_ADMIN, SYSTEM_ROLE.DATA_ADMINISTRATOR]
+        }
+      ]
+    };
+  }),
+  updateAlert()
+];
+
+PUT.apiDoc = {
+  description: 'Update an alert by its id.',
+  tags: ['alerts'],
+  security: [
+    {
+      Bearer: []
+    }
+  ],
+  parameters: [
+    {
+      in: 'query',
```

Comment:
`in: 'path'`

and should be `required: true`

Also, should be an `'integer'`